src/main/java/uk/ac/ebi/eva/pipeline/jobs/steps/tasklets/LoadStatisticsTasklet.java

```diff
@@ -225,7 +228,22 @@ private void loadSourceStats(VariantDBAdaptor variantDBAdaptor, URI sourceStatsU
         VariantSourceStats variantSourceStats = sourceParser.readValueAs(VariantSourceStats.class);
 
         // Store source statistics in Mongo
-        variantDBAdaptor.getVariantSourceDBAdaptor().updateSourceStats(variantSourceStats, null);
+        updateSourceStats(variantSourceStats);
```

Comment:
I understand that this sidesteps the OpenCGA issue but let's mark this with a TODO to indicate that this is a temporary measure. I also propose that we create a ticket to update OpenCGA MongoDB in the future and reference this line of code in that ticket as a candidate for remediation.

Reply:
Done, [ticket](https://www.ebi.ac.uk/panda/jira/browse/EVA-2025) created